src/exchangeRate.ts

```diff
@@ -0,0 +1,58 @@
+import { asNumber, asObject, asOptional, asString } from 'cleaners'
+
+import config from '../config.json'
+
+export interface CurrencyOption {
+  address: string
+  currencyName: string
+  coinCapCurrencyName: string
+}
+
+const asPriceUsd = asObject({
+  priceUsd: asString
+})
+
+const asCoinCapResponse = asObject({
+  data: asOptional(asPriceUsd),
+  error: asOptional(asString),
+  timestamp: asNumber
+})
```

Comment:
This should change into asRatesServerResponse.

Reply:
Updated per comment.